### DISCUSSION TITLE
Unable to import

### DISCUSSION THREAD

**Discussion by skywalkerlw** [pointfreeco/swift-dependencies#139] [General]
### Description

I created an iOS demo and then imported this library.
I thought it should be enough straightforward to use it, then I found  Xcode can not recognize this library at all.
Anything I missing ?

![image](https://github.com/pointfreeco/swift-dependencies/assets/2090084/cd3cdfa8-3641-49f4-b99c-77254ae54b03)

[Demo.zip](https://github.com/pointfreeco/swift-dependencies/files/13401566/Demo.zip)

Check the attached demo project.

### Checklist

- [ ] I have determined whether this bug is also reproducible in a vanilla SwiftUI project.
- [X] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-dependencies/issues) or [discussion](https://github.com/pointfreeco/swift-dependencies/discussions).

### Dependencies version information

1.1

### Destination operating system

iOS 17

### Xcode version information

15

### Swift Compiler version information

```shell
swift-driver version: 1.87.1 Apple Swift version 5.9 (swiftlang-5.9.0.128.108 clang-1500.0.40.1)
Target: arm64-apple-macosx14.0
```

**Comment by mbrandonw** [comment]
Hi @skywalkerlw, you did not link the Dependencies library to your target, and that is why you can't import it:

<img width="717" alt="Screenshot 2023-11-18 at 7 19 04 PM" src="https://github.com/pointfreeco/swift-dependencies/assets/135203/56b0b04d-c550-45de-8165-e6e69c8004fe">

Once you do that it works just fine:

<img width="761" alt="image" src="https://github.com/pointfreeco/swift-dependencies/assets/135203/7edc5cf3-af7d-4e37-ad3f-177ad110f494">

Since this isn't an issue with the library I am going to convert it to a discussion.
